CHANGELOG.md

```diff
@@ -12,6 +12,18 @@
 
 ---
 
+[//]: # (START/v1.9.0)
+# v1.9.0
+
+## Features
+  * Operator can be used with 1Password Service Accounts now. {#160}
```

Comment:
`[nit]`

```suggestion
  * Enable the Operator to authenticate to 1Password using service accounts. {#160}
```
or
```suggestion
  * Add service account support for the Operator. {#160}
```

Using the word `now` in a changelog message is not that elegant.